### PR TITLE
Problem: tasks index page is missing

### PR DESCRIPTION
**Pull request by maslowalex**
Screenshot 2023-06-10 at 14 39 56

### CODE REVIEW

File: assets/src/components/Tasks.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+function TasksIndex() {
+  const [tasks, setTasks] = useState<State>([])
+
+  useEffect(() => {
+    function doFetchTasks() {
+      fetchTasks().then(response => {
+        if ('tasks' in response) {
+          setTasks(response.tasks)
+        } else {
+          // Handle error
```
Comment (cognivore) [inline assets/src/components/Tasks.tsx:16]: The code LGTM, I just need to check one thing here.

We should return Result as Uptight Result and just print the assertion error + trace information in errors by default.

Make sure we're doing that.

Then, when we're going to be making the errors prettier, we'll have a dictionary function that takes an error and "translates" it depending on who is looking at the error. If it's the user, it's gonna be a "translation" from English to English, perhaps, if it's a developer, it's gonna be an identity function.

**Review by cognivore — APPROVED** [review]
LGTM, but see the inline comment.

File: assets/src/components/Tasks.tsx (hunk 1 above)
Comment (maslowalex) [reply assets/src/components/Tasks.tsx:16]: Sure, it *is* working like that for commands submission, but here the only error I expect to see is `Unauthorized` (when session will expire) and we want to just `window.location('/login')` here for example. In this case there will no be any Uptight Result returned.

**Comment by cognivore** [comment]
With Uptight, we can't expect any errors, ever!

The reason is that since we're treating errors as faults, in presence of
which we want to keep operating.

That's why, every fetch should return an ok tuple with HTTP status and
payload encoding a Result structure as JSON.

That's the point of the library and we have to always return result from
Phoenix.

**Pull request merged**